csvkit/utilities/in2csv.py

```diff
@@ -20,6 +23,12 @@ class In2CSV(CSVKitUtility):
     override_flags = ['f']
 
     def add_arguments(self):
+        def option_parser(bytestring):
+            if six.PY2:
+                return bytestring.decode(sys.getfilesystemencoding())
+            else:
+                return bytestring
```

Comment:
@onyxfish I'm not fond of this new `option_parser` method but I don't know how else to do this across Python 2 and 3.